src/lib/device-types/build-info-facade.ts

```diff
@@ -14,6 +15,34 @@ export const getIsIgnored = memoizee(
 	{ promise: true, preFetch: true, maxAge: BUILD_PROPERTY_CACHE_EXPIRATION },
 );
 
+export const getLogoUrl = memoizee(
+	async (
+		normalizedSlug: string,
+		buildId: string,
+	): Promise<string | undefined> => {
+		if (!FILES_HOST) {
+			return;
+		}
+		try {
+			const logoKey = getImageKey(normalizedSlug, buildId, 'logo.svg');
+			const hasLogo = await fileExists(logoKey);
+			if (!hasLogo) {
+				return;
+			}
+
+			const encodedLogoPath = getImageKey(
+				normalizedSlug,
+				encodeURIComponent(buildId),
+				'logo.svg',
+			);
+			return `https://${FILES_HOST}/${encodedLogoPath}`;
```

Comment:
Shouldn't this be a signed URL? What if the bucket or object is private?

Reply:
This PR indeed doesn't handle private device types.
Need to re-discuss this with @afitzek but I would like to unblock this for the public ones at least, so that we don't have to manually add DT logos to the dashboard sooner than later.
See: https://github.com/balena-io/balenacloud-dashboard/issues/194 (since 2016)